README.md

```diff
@@ -82,15 +84,17 @@ docker run --name <container_name> -v </path/to/config.json>:/var/www/assets/con
 Make sure `</path/to/config.json>` is absolute path. To read more about configuration file go to [environment configuration section](#environment-configuration).   
 The application will be served at `http://localhost:<port>`.
 
+The second (and more complex) way to serve the application is via local development server. Go straight to [Development](#development) section if that's your case.
+
 ## Development
```

Comment:
This section is dedicated to development, I suggest removing Build step, or moving it to Deployment section.

Reply:
I also suggest renaming Local run section, to Deployment or Build and Run

Reply:
I may seem like Environment Configuration subsection belongs to Local run section, but since it is also used for development setup. How about moving it to a full-fledged section and add links when it's needed.